### PR TITLE
Added crossbrowser and fixed tests in IE

### PR DESCRIPTION
**Pull request by nealstewart**

### CODE REVIEW

File: test/tests/channeling.js [hunk 1]
```diff
@@ -141,26 +127,42 @@ describe('Channeling and exorcism', function likethis() {
...
-	it('should (eventually) exorcize on outerHTML', function(done) {
-		var container = document.createElement('div');
-		var elm = getSpiritElm();
-		container.appendChild(elm);
-		this.sandbox.appendChild(container);
-		elm.outerHTML = '<div></div>';
-		sometime(function later() {
-			expect(hasSpirit(elm)).toBe(false);
-			done();
+	if (!gui.Client.isExplorer9 || gui.Client.isExplorer10) {
```
Comment (wiredearp) [inline test/tests/channeling.js]: If it's not IE9 or if it's IE10. Isn't that covered by not-IE9 to begin with?

File: test/tests/channeling.js [hunk 2]
```diff
@@ -141,26 +127,42 @@ describe('Channeling and exorcism', function likethis() {
...
-	it('should (eventually) exorcize on outerHTML', function(done) {
-		var container = document.createElement('div');
-		var elm = getSpiritElm();
-		container.appendChild(elm);
-		this.sandbox.appendChild(container);
-		elm.outerHTML = '<div></div>';
-		sometime(function later() {
-			expect(hasSpirit(elm)).toBe(false);
-			done();
+	if (!gui.Client.isExplorer9 && !gui.Client.isExplorer10) {
```
Comment (wiredearp) [inline test/tests/channeling.js]: Does it work in IE11?

**Comment by wiredearp** [comment]
:v:

File: test/tests/channeling.js (hunk 2 above)
Comment (nealstewart) [inline test/tests/channeling.js]: Yep.

**Pull request merged**